apply_dimension.json

```diff
@@ -74,7 +74,7 @@
         }
     ],
     "returns": {
-        "description": "A data cube with the newly computed values for the specified. The resolution and the number of dimensions are the same as for the original data cube.",
+        "description": "A data cube with the newly computed values. All dimensions stay the same, except for the dimensions specified in corresponding parameters. There are three cases how the data cube changes:\n\n1. The source dimension **is** the target dimension:\n   * The (number of) dimensions have not changed.\n   * The source dimension properties name, type and reference system have not changed.\n   * The dimension labels and the resolution are preserved when the number of pixel values in the source dimension is equal to the number of values computed by the process. The other case is described below.\n2. The source dimension **is not** the target dimension and the latter **exists**:\n   * The number of dimensions decreased by one as the source dimension has been dropped.\n   * The target dimension properties name, type and reference system have not changed.\n   * The resolution has changed, the number of dimension labels is equal to the number of values computed by the process and the dimension labels are incrementing integers starting from zero\n3. The source dimension **is not** the target dimension and the latter **does not exist**:\n   * The number of dimensions have not changed, but the source dimension has been replaced with the target dimension.\n   * The target dimension has the specified name and the type `other`. The reference system has not been changed.\n   * The resolution has changed, the number of dimension labels is equal to the number of values computed by the process and the dimension labels are incrementing integers starting from zero\n\nFor all three cases except for the exception in the first case, the resolution has changed, the number of dimension labels is equal to the number of values computed by the process and the dimension labels are incrementing integers starting from zero.",
```

Comment:
"description": "A data cube with the newly computed values. All dimensions stay the same, except for the dimensions specified in corresponding parameters. There are three cases how the data cube changes:
\n
\n1. The source dimension **is** the target dimension:
\n   * The (number of) dimensions remain unchanged.
\n   * The source dimension properties name, type and reference system remain unchanged.
\n   * The dimension labels and the resolution are preserved when the number of pixel values in the source dimension is equal to the number of values computed by the process. The other case is described below.

\n2. The source dimension **is not** the target dimension and the latter **exists**:
\n   * The number of dimensions decreases by one as the source dimension is dropped.
\n   * The target dimension properties name, type and reference system remain unchanged.
\n   * The resolution changes, the number of dimension labels is equal to the number of values computed by the process and the dimension labels are incrementing integers starting from zero

\n3. The source dimension **is not** the target dimension and the latter **does not exist**:
\n   * The number of dimensions do not changed, but the source dimension is replaced with the target dimension.
\n   * The target dimension has the specified name and the type `other`. The reference system is not changed.
\n   * The resolution changes, the number of dimension labels is equal to the number of values computed by the process and the dimension labels are incrementing integers starting from zero
\n
\nFor all three cases except for the exception in the first case, the resolution changes, the number of dimension labels is equal to the number of values computed by the process and the dimension labels are incrementing integers starting from zero.

Reply:
@przell I guess "do not changed" should be "remain unchanged"